Makefile

```diff
@@ -63,7 +63,9 @@ install-tools:
 			github.com/client9/misspell/cmd/misspell \
 			github.com/golangci/golangci-lint/cmd/golangci-lint \
 			github.com/google/addlicense \
-			github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen
+			go.opentelemetry.io/collector/cmd/mdatagen \
+			github.com/google/googet/goopack \
```

Comment:
Do you do a googet package of this repo? If not you probably don't need this

Reply:
done